modules/dxp/apps/commerce-avalara/commerce-avalara-lang/src/main/resources/content/Language.properties

```diff
@@ -0,0 +1,9 @@
+account-number=Account Number
+avalara=Avalara
+avalara-tax-rate-description=Avalara Type Rate Description
+commerce-avalara-connecto-configuration-name=Commerce Avalara Connector Configuration Name
+company-code=Company Code
+could-not-verify-the-connection-the-provided-credentials-are-not-correct=Could not verify the connection, the provided credentials are not correct
```

Comment:
@riccardo-alberti still missing a period at the end of correct.

Reply:
missed that.. 👍